cpu/msp430_common/include/cpu.h

```diff
@@ -7,20 +7,25 @@
  * directory for more details.
  */
 
-#ifndef CPU_H
-#define CPU_H
-
 /**
  * @defgroup    cpu_msp430_common TI MSP430
  * @ingroup     cpu
  * @brief       Texas Instruments MSP430 specific code
-
-<h2>First steps</h2>
-\li See the <a href="../manual/index.html">manual</a> for toolchain and ide setup
-
+ *
+ * ## First steps
+ *
+ * See the <a href="../manual/index.html">manual</a> for toolchain and ide
```

Comment:
I have no idea what file this is supposed to point to. It doesn't work on https://api.riot-os.org or https://riot-os.org/api

Reply:
me neither and looking at the [wiki](https://github.com/RIOT-OS/RIOT/wiki/Family:-MSP430) the links are broken.

Reply:
What about pointing to the [TI website](http://www.ti.com/tool/msp430-gcc-opensource#Technical%20Documents) ?
I have no experience with MSP430 so maybe others may have a better opinion on that.